polkadot/runtime/parachains/src/hrmp.rs

```diff
@@ -836,6 +836,34 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// TODO write docs
```

Comment:
Just FYI we have a CI job that scans for the string `FAIL-CI` to remind of todos 😄